examples/azure_ad.py

```diff
@@ -0,0 +1,34 @@
+from azure.identity import DefaultAzureCredential
```

Comment:
We can update this to use the real auth helper function now:

```python
from azure.identity import DefaultAzureCredential, get_bearer_token_provider

token_provider = get_bearer_token_provider(DefaultAzureCredential(), "https://cognitiveservices.azure.com/.default")


client = AzureOpenAI(
    api_version=api_version,
    azure_endpoint=endpoint,
    azure_ad_token_provider=token_provider,
)
```

Reply:
Yes, we released a new version of azure-identity that has the [bearer token provider](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/identity/azure-identity/CHANGELOG.md#features-added-1) yesterday.